frontend/src/components/permissions/helpers/get-permissions-name/get-permissions-name.ts

```diff
@@ -0,0 +1,7 @@
+import { IUserWithPermissions } from 'common/interfaces';
```

Comment:
rename to `frontend/src/components/permissions/helpers/get-permissions-name/get-permissions-name.helper.ts`